setup.py

```diff
@@ -42,7 +41,7 @@
     download_url = 'https://github.com/buttercms/buttercms-python/tarball/0.1',
     py_modules=['butter_cms'],
     install_requires=install_requires,
-    keywords = ['foo', 'bar', 'baz'],
+    keywords = ['buttercms', 'sdk', 'cms', 'api'],
     classifiers=[
```

Comment:
Also, kwargs `author` and `author_email`.

Reply:
Sure, I'll double check if it's compatible with older version of python and django framework. 

Regarding dev status, I think we can keep the development status for Alpha or move to Beta. It might make sense to move to 5 after tests are fixed. 

I'm not sure about the author - maybe @ViolanteCodes can help us with that.

Reply:
> Sure, I'll double check if it's compatible with older version of python and django framework.
> 
> Regarding dev status, I think we can keep the development status for Alpha or move to Beta. It might make sense to move to 5 after tests are fixed.
> 
> I'm not sure about the author - maybe @ViolanteCodes can help us with that.

I think Author should just be `ButterCMS` and author email `support@buttercms.com`

Reply:
Tested compatibility with both 2.7.18 python and django - working correctly  - changing only the author information